sootup.java.bytecode/src/main/java/sootup/java/bytecode/interceptors/Aggregator.java

```diff
@@ -191,22 +191,24 @@ public void interceptBody(@Nonnull Body.BodyBuilder builder, @Nonnull View view)
         Stmt newStmt;
 
         final ReplaceUseStmtVisitor replaceVisitor = new ReplaceUseStmtVisitor(val, aggregatee);
-        // FIXME: this try-catch is an awful hack for "ValueBox.canContainValue" -> try to determine
+        // TODO: this try-catch is an awful hack for "ValueBox.canContainValue" -> try to determine
         // a replaceability earlier!
         try {
           replaceVisitor.caseAssignStmt(assignStmt);
           newStmt = replaceVisitor.getResult();
         } catch (ClassCastException iae) {
-          newStmt = null;
+          continue;
```

Comment:
log warning?

Reply:
nope - same as above (hack to not existing  "isReplaceableByThisValue()")